Doc/library/typing.rst

```diff
@@ -666,6 +666,13 @@ The module defines the following classes, functions and decorators:
 
    .. versionadded:: 3.6
 
+.. class:: AsyncContextManager(Generic[T_co])
+
+   An ABC with async abstract :meth:``__aenter__`` and :meth:``__aexit__``
```

Comment:
Sorry for giving you a wrong advice, it actually should be `` :meth:`__aiter__` `` (single backticks). You can also try `make html` and check that they are rendered correctly in docs.

Reply:
Should have caught it myself. Fixed it and built it and works as expected. :tada: